pysembench/core.py

```diff
@@ -55,7 +66,7 @@ def __init__(
             self.sembench_config_path or self.sembench_data_location
         )
         self.sembench_config_file_name = (
-            self.sembench_config_file_name or "sembench.json"
+            self.sembench_config_file_name or "sembench.yaml"
         )
 
         if not self.sembench_config_path.endswith(
```

Comment:
this currently leads to 
`AttributeError: 'NoneType' object has no attribute 'endswith'`
and a stack-trace dump that is not really helping the end user

better approach is to have proper assertions during __init__ with clear end-user messages on how to resolve